src/components/pages/ChildDashboard/RenderChildDashboard.js

```diff
@@ -15,7 +15,7 @@ const RenderChildDashboard = props => {
   const [modalVisible, setModalVisible] = useState(false);
 
   const handleAcceptMission = e => {
-    push('/gameplay/mission/read');
+    push('/gameplay/gamemode');
```

Comment:
Nice route change

Reply:
gameplay/gamemode page will then point to gameplay/mission/read ad a sibling to gamemode. Nice work stitching that in there.